school_student.rb

```diff
@@ -0,0 +1,12 @@
+require './school_person'
+
+class Student < Person
+  def initialize(classroom)
+    super
```

Comment:
- [ ] Constructor extends parent's constructor by adding @classroom and a parameter for it like: `def initialize(age, classroom, name = "Unknown", parent_permission = true)`. Please implement it.